database/index.js

```diff
@@ -1,6 +1,11 @@
 const fs = require('fs');
 const path = require('path');
-const { compose, last, head } = require('ramda');
+const knex = require('knex');
+const { compose, last, head, tap } = require('ramda');
+const { getConfig } = require('../config');
+
+// lazyPromiseSeries :: [PromiseLike (a)] -> Promise ([a])
```

Comment:
what's `PromiseLike`?

Reply:
Something that is compatible with the Promise/A+ interface but isn't a native `Promise` (e.g. the `knex.QueryBuilder`)